docs/en_US/errors/MRTI-678-1005.md

```diff
@@ -0,0 +1,24 @@
+# Error Code: 678-1005
+**Applies to:** Wii U
+
+---
+
+***Note:*** This error relates to the Martini patches, which are now deprecated.
+
+The patcher could not find an unmodified copy of the Miiverse applet. Either you have manually modified it, it has
+become corrupted, or you have a version too old or too new for the patcher to recognise.
+
+- If you are on a system version older than 5.1.0, perform a system update, install
+  [Tiramisu or Aroma](https://wiiu.hacks.guide), and try again.
+- Try disabling any content replacement plugins (SDCafiine etc.) that can change which files are read by the patcher.
+- If you have manually modified your Miiverse applet, reinstall a clean copy from NUS or disc. Instructions for this
```

Comment:
You know I get hyper worried about things, and wording scares me

Are you against saying something like "reinstall from a backup" instead of directly mentioning NUS here? Saying just "backup" is ambiguous enough to where it could mean either a user made backup of their own copy, or. You know. How pirates refer to "backups". That way we can at least argue we didn't push people in the "go look on NUS" direction?

(only gonna comment this one time despite it happening twice)

Reply:
you know how I feel about system titles and the NUS, but I'll defer to you on this one.

I wrote a little bit on the fence about the "backup" phrasing (since Martini is an app that creates said backups) but then I remembered that this sentence starts with "If you have manually modified your Miiverse applet" so I think I'm cool with that change

Reply:
I know how you feel, I'm not trying to discount or invalidate that, you know I try to just keep everything here in the up-and-up whenever I can

Sounds good, then